app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+  "In close contact without symptoms": "Em contacto próximo sem sintomas",
+  "Close contact with someone who has tested positive": "Contacto próximo com alguém que testou positivo",
+  "Tested negative": "Negativos",
+  "Map": "Mapa",
+  "API": "API",
+  "I agree to set a cookie so I can change my health condition later e.g. to report recovery/getting sick": "Concordo em utilizar uma cookie para poder alterar o meu estado de saúde mais tarde, por exemplo para relatar situação de recuperação ou doença",
+  "required": "obrigatório",
+  "recommended": "recomendado",
+  "Below is your personal health information.": "Abaixo encontra-se a sua informação pessoal.",
+  "If you want to register another person on this computer, save or write down your personal link": "Se pretender registar outra pessoa neste computador, guarde ou anote o seu endereço pessoal",
+  "Your personal health information is cleared from this computer.": "A sua informação pessoal foi eliminada deste computador.",
+  "You can now use this computer for another user.": "Pode agora utilizar este computador para outra pessoa.",
+  "We stored your data in a cookie so come back later to update your status": "Guardámos os seus dados numa cookie, no seu computador, para que possa voltar mais tarde e atualizar o seu estado",
+  "If you use this computer with multiple users, please use this link to update your status.": "Se utilizar este computador com multiplos utilizadores, por favor utilize este endereço para atualizar o seu estado.",
+  "Write it down somewhere or save it on your computer.": "Anote-o ou guarde no seu computador.",
+  "The link has been copied.": "O endereço foi copiado.",
+  "clear this form": "limpe os dados deste formulário",
+  "https://who.int": "https://covid19.min-saude.pt/",
+  "To WHO.int": "Direção Geral de Saúde (DGS).",
+  "Are you from": "Vive em",
+  "Norway": "Norway",
+  "The Netherlands": "The Netherlands",
+  "Slovakia": "Slovakia",
+  "United Kingdom": "The United Kingdom",
+  "March 16": "16 de Março",
+  "March 17": "17 de Março",
+  "March 23": "23 de Março",
+  "at": "às",
+  "Nausea or vomiting": "Náusea ou vómito",
+  "Have you been diagnosed with any of the following conditions?": "Foi diagnosticado/a com alguma das seguintes condições?",
+  "Diabetes mellitus, Hypertension, Ischemic heart disease, Asthma, Chronic lung disease, Chronic kidney disease": "Diabetes, Hipertensão, Doença arterial coronária, Asma, Doença pulmonar crónica, Doença renal crónica",
+  "I am currently:": "Encontro-me atualmente",
+  "Not in isolation": "Não estou em isolamento",
+  "In isolation due to a recent international travel": "Em isolamento devido a uma recente viagem internacional",
+  "In isolation due to contact with an individual who has been exposed to the coronavirus.": "Em isolamento devido a contacto com alguém exposto ao coronavirus",
+  "Cigarette smoking habits:": "Hábitos de fumador",
```

Comment:
Is this correctly translated? If so can you add it to the PR?
```"In voluntary isolation to avoid potentially spreading or catching COVID-19": "Em isolamento voluntário para evitar uma potencial propagação ou contração do COVID-19",
"March 28": "23 de Março",```